app/Filament/Resources/Budgeting/ExpenseResource/Actions/QuickExpenseAction.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Filament\Resources\Budgeting\ExpenseResource\Actions;
+
+use App\Models\Expense;
+use App\ValueObjects\Money;
+use Filament\Forms\Components\Hidden;
+use Filament\Forms\Components\TextInput;
+use Filament\Support\RawJs;
+use Filament\Tables\Actions\CreateAction;
+
+class QuickExpenseAction extends CreateAction
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this
+            ->label('New')
+            ->icon('heroicon-s-plus')
+            ->form([
+                TextInput::make('description')
+                    ->required()
+                    ->maxLength(255),
+                TextInput::make('amount')
+                    ->required()
+                    ->mask(RawJs::make('$money($input, \',\', \'.\')'))
+                    ->prefix('Rp.')
+                    ->dehydrateStateUsing(fn (?string $state) => Money::makeFromFilamentMask($state)->value),
+                Hidden::make('expense_id')
+                    ->default(fn (Expense $record): int => $record->id),
```

Comment:
remove this line instead you add it in the action() you can get the Expense model as $record right? why not to avoid this hidden for security reason?